electron-app/src/services/rotkehlchen-api.ts

```diff
@@ -416,12 +416,17 @@ export class RotkehlchenApi {
       .then(this.handleResponse);
   }
 
-  queryBalancesAsync(): Promise<AsyncQuery> {
+  queryBalancesAsync(
+    ignoreCache: boolean = false,
+    saveData: boolean = false
+  ): Promise<AsyncQuery> {
     return new Promise<AsyncQuery>((resolve, reject) => {
       this.axios
         .get<ActionResult<AsyncQuery>>('/balances/', {
           params: {
-            async_query: true
+            async_query: true,
+            ignore_cache: ignoreCache ? true : undefined,
```

Comment:
Why undefined? This is a boolean value. Just use the `ignoreCache` and `saveData` booleans respectively. Providing undefined to the backend is an error.

Reply:
I think this case it just actually doesn't pass the parameter, it doesn't literally send `ignoreCache=undefined`. It's the same way kelsos uses it throughout.

Reply:
Is that true @kelsos? Is this a way to not pass the parameter? Or does it end up sending a `null`  value?

But still why do it like that? It's a boolean value. Can be either true or false.

Reply:
It's effectively a way to not pass the parameter (if you check the logs when this happens there shouldn't be a parameter passed).

Reply:
Still I prefer explicitness and since this is a true/false value it's easy to just pass the desired value or not.

Reply:
Yes if `true` it will include `ignore_cache` in the query parameter. If it is `undefined` it will not appear in the URL. No `null` is a different value than `undefined`. `undefined` has no value, `null` has value, though both evaluate to `falsy`